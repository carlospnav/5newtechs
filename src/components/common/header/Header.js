import styled from 'styled-components';
import { flexGroup } from "theme/utils"

export default styled.header`
  ${flexGroup("column", "center", "center")}
  height: ${props => props.main ? '660px' : '200px' }
  width: 100%;
  z-index: 1;
`
