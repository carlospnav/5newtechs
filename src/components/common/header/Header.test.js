import { shallow } from "enzyme"
import Header from "./Header"
import "jest-styled-components"

describe("<Header>", () => {
  it("should render a main Header properly.", () => {
    const wrapper = shallow(<Header main/>)
    expect(wrapper).toMatchSnapshot()
  })

  it("should render a regular Header properly.", () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})