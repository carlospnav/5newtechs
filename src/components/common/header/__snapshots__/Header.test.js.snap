// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> should render a main Header properly. 1`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 660px;
  width: 100%;
  z-index: 1;
}

<header
  className="c0"
/>
`;

exports[`<Header> should render a regular Header properly. 1`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 200px;
  width: 100%;
  z-index: 1;
}

<header
  className="c0"
/>
`;
